doc/manuals/admin/sanity_check.md

```diff
@@ -30,6 +30,9 @@ curl --header 'Accept: application/json' localhost:1026/version
     "compile_time" : "Tue Sep 8 16:56:16 CEST 2015",
     "compiled_by" : "fermin",
     "compiled_in" : "debvm"
+    "release_date" : "Tue Sep 8 16:56:16 CEST 2015",
+    "doc" : "https://fiware-orion.readthedocs.org/en/master/"
+
```

Comment:
Please remove this added empty line.

Reply:
I have also take the opportunity to update some other tokens in this example, e.g.

version -> 1.8.0-next
date -> to the current date tooday :)

Reply:
Fixed in 957bf89